config/overlays/helm/exclude-role.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
```

Comment:
Why this file is needed?

Reply:
This is because a helm value that enables this new feature has been added- since the controller for the `Flagd` resource requires additional rbac permissions (creating, deleting, updating Services and Ingresses), they will only be created if the related helm value is set to true. Therefore, a customized rbac manifest that makes use of this helm value has been added to the chart, and the default role file created by kustomized is replaced with that one